inc/tainetworkif.h

```diff
@@ -511,6 +511,30 @@ typedef enum _tai_network_interface_attr_t
      */
     TAI_NETWORK_INTERFACE_ATTR_NOTIFY,
 
+    /**
+     * @brief The current post-VOA RX input power in the provisioned channel in dBm
+     *
+     * @type #tai_float_t
+     * @flags READ_ONLY
+     */
+    TAI_NETWORK_INTERFACE_ATTR_CURRENT_POST_VOA_PROVISIONED_CHANNEL_POWER,
+
+    /**
+     * @brief The RX power low warning threshold in dBm
+     *
+     * @type #tai_float_t
+     * @flags READ_ONLY
```

Comment:
@samisaab Can't we change this threshold?

How do we know when the rx power fall below this threshold?
Do we need to add an attribute to get an alarm for this?

Reply:
Yes.  I'll take a look at implementing the alarm and let you know if I have questions.